ui-tests/src/test/java/core/GitBlamerUITest.java

```diff
@@ -0,0 +1,287 @@
+
+        assertColumnsOfRowLoremIpsum(table.getRowAs(3, BlamesTableRow.class), commits.get("LoremIpsum"));
+        assertColumnsOfRowLoremIpsum(table.getRowAs(4, BlamesTableRow.class), commits.get("LoremIpsum"));
+        assertColumnsOfRowLoremIpsum(table.getRowAs(5, BlamesTableRow.class), commits.get("LoremIpsum"));
+        assertColumnsOfRowLoremIpsum(table.getRowAs(6, BlamesTableRow.class), commits.get("LoremIpsum"));
+
+        assertColumnsOfTest(table.getRowAs(0, BlamesTableRow.class), commits.get("Test"));
+        assertColumnsOfTest(table.getRowAs(1, BlamesTableRow.class), commits.get("Test"));
+        assertColumnsOfTest(table.getRowAs(2, BlamesTableRow.class), commits.get("Test"));
+        assertColumnsOfTest(table.getRowAs(3, BlamesTableRow.class), commits.get("Test"));
+    }
+
+    private void assertColumnsOfTest(final BlamesTableRow row, final String commit) {
+        assertThat(row.getAuthor()).isEqualTo("Git SampleRepoRule");
+        assertThat(row.getEmail()).isEqualTo("gits@mplereporule");
+        assertThat(row.getFileName()).isEqualTo("Test.java");
+        assertThat(row.getCommit()).isEqualTo(commit);
+        assertThat(row.getAge()).isEqualTo(1);
+    }
+
+    private void assertColumnsOfRowBob(final BlamesTableRow row, final String commit) {
+        assertThat(row.getAuthor()).isEqualTo("Alice Miller");
+        assertThat(row.getEmail()).isEqualTo("alice@miller");
+        assertThat(row.getFileName()).isEqualTo("Bob.java");
+        assertThat(row.getCommit()).isEqualTo(commit);
+        assertThat(row.getAge()).isEqualTo(1);
+    }
+
+    private void assertColumnsOfRowLoremIpsum(final BlamesTableRow row, final String commit) {
+        assertThat(row.getAuthor()).isEqualTo("John Doe");
+        assertThat(row.getEmail()).isEqualTo("john@doe.de");
+        assertThat(row.getFileName()).isEqualTo("LoremIpsum.java");
+        assertThat(row.getCommit()).isEqualTo(commit);
+        assertThat(row.getAge()).isEqualTo(1);
+    }
+
+    private void assertColumnHeader(final BlamesTable table) {
+        assertThat(table.getHeaders()).containsExactly(DETAILS, FILE, AGE, AUTHOR, EMAIL, COMMIT, ADDED);
+    }
+
+    private Build generateFreeStyleJob(final GitRepo repo) {
+        repo.transferToDockerContainer(host, port);
+        job.useScm(GitScm.class)
+                .url(repoUrl)
+                .credentials(USERNAME);
+
+        addRecorder((FreeStyleJob) job);
+        job.save();
+
+        return job.startBuild().waitUntilFinished();
+    }
+
+    private Build generateWorkflowJob(final GitRepo repo) {
+        repo.transferToDockerContainer(host, port);
+
+        // workflowJob.sandbox.check();
+        /* workflowJob.useScm(GitScm.class)
+                .url(repoUrl)
+                .credentials(USERNAME); */
+        workflowJob.setJenkinsFileRepository(repoUrl, USERNAME);
+        workflowJob.script.set("node {\n"
```

Comment:
`workflowJob.script.set` das gibt es nur falls nicht das Repository verwendet wird. Am besten mal selbst im Jenkins UI anschauen. 

D.h. das Script wird nicht hier gesetzt sondern als Datei in Git abgelegt